js/data/buffer.js

```diff
@@ -65,29 +64,19 @@ function Buffer(options) {
 }
 
 // Start Buffer1 compatability code
-Buffer.prototype = {
-
-    get pos() { return this.length * this.itemSize; },
-    get index() { return this.length; },
-
-    get arrayType() { return this.type; },
-
-    get array() { return this.arrayBuffer; },
-    set array(value) {
-        assert(value === null);
-        this.arrayBuffer = null;
-    },
 
+Buffer.prototype = {
```

Comment:
We should use the same `Buffer.prototype.bind = ` syntax here and add a JSdoc comment